packages/env/README.md

```diff
@@ -11,6 +11,7 @@
 ## What you get with this project
 
 - `@theforeman/env/babel` - Adds theforeman babel dev configuration to your project.
+- `@theforeman/env/jest` - Adds theforeman jest configuration to your project.
```

Comment:
Notice line 16.
Wondering if we should call it `@theforeman/env/test` so one day we can replace `jest` with something else for testing.